scene/2d/line_builder.cpp

```diff
@@ -170,7 +172,7 @@ void LineBuilder::build() {
 		if (texture_mode == Line2D::LINE_TEXTURE_TILE) {
 			uvx0 = 0.5f / tile_aspect;
 		}
-		new_arc(pos0, pos_up0 - pos0, -Math_PI, color0, Rect2(0.f, 0.f, 1.f, 1.f));
+		new_arc(pos0, pos_up0 - pos0, -Math_PI, color0, Rect2(0.f, 0.f, fmin(uvx0 * 2, 1.f), 1.f));
```

Comment:
I'm not sure about this line... normally, in tile mode, a round cap should look like this:
![image](https://user-images.githubusercontent.com/1311555/42776599-66b9bff0-892f-11e8-887d-fd54fa6f3088.png)
I'll have to check if your changes affect this, unless you can test earlier.
Hmm...
I think I understand better what you fixed:
![image](https://user-images.githubusercontent.com/1311555/42776810-1f397476-8930-11e8-8adf-945f10d1f7fd.png)